templates/smtp.rb

```diff
@@ -1,9 +1,20 @@
+if ENV['SMTP_PROVIDER'] == 'mandrill'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.